exercises/static/exercises/drone_cat_mouse/web-template/launch/drone_cat_mouse.launch

```diff
@@ -0,0 +1,108 @@
+
+  <!-- The following lines simulate the world in Gazebo. The physic engine properties
+        are set up in the file "basic_crazyflie.world" file -->
+  <env name="GAZEBO_MODEL_PATH" value="${GAZEBO_MODEL_PATH}:$(find rotors_gazebo)/models"/>
+  <env name="GAZEBO_RESOURCE_PATH" value="${GAZEBO_RESOURCE_PATH}:$(find rotors_gazebo)/models"/>
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <arg name="world_name" value="/RoboticsAcademy/exercises/drone_cat_mouse/web-template/drone_cat_mouse.world" />
+    <!-- <arg name="x" value="14.96"/>
+    <arg name="z" value="0.3"/>
+    <arg name="y" value="1.57"/> -->
+    <arg name="debug" value="$(arg debug)" />
+    <arg name="paused" value="$(arg paused)"/>
+    <arg name="use_sim_time" value="$(arg use_sim_time)"/>
+    <arg name="headless" value="$(arg headless)"/>
+    <arg name="gui" value="$(arg gui)" />
+    <arg name="verbose" value="$(arg verbose)"/>
+  </include>
+  
+  <group ns="$(arg mav_name)_1">
+  <!-- CRAZYFLIE_1 -->
+    <include file="$(find rotors_gazebo)/launch/spawn_mav_crazyflie.launch">
+      <arg name="mav_name" value="$(arg mav_name)" />
+      <arg name="namespace" value="$(arg mav_name)_1" />
+      <arg name="model" value="$(find rotors_description)/urdf/mav_generic_odometry_sensor.gazebo" />
+      <arg name="enable_logging" value="$(arg enable_logging)" />
+      <arg name="enable_ground_truth" value="$(arg enable_ground_truth)" />
+      <!-- As for the Mellinger and Position controllers in the "crazyflie2_hovering_example.launch" file -->
+      <arg name="enable_internal_model_controller" value="$(arg enable_internal_model_controller)" />
+      <arg name="log_file" value="$(arg mav_name)_1"/>
+      <!-- Set the initial position -->
+      <arg name="x" value="-0.5"/>
+      <arg name="y" value="-0.5"/>
+    </include>
+   <!-- The Crazyflie position controller -->
+   <node name="position_controller_node" pkg="rotors_control" type="position_controller_node" output="screen">
+      <!-- Enabling internal model controller-->
+      <param name="enable_internal_model_controller" value="$(arg enable_internal_model_controller)"  />
+      <!-- Crazyflie file parameters used within the Internal Model Controller controller -->
+      <rosparam command="load" file="$(find rotors_gazebo)/resource/crazyflie_parameters.yaml" />
+      <rosparam command="load" file="$(find rotors_gazebo)/resource/$(arg mav_name).yaml" />
+      <!-- Loading Internal Model Controller's parameters -->
+      <rosparam command="load" file="$(find rotors_gazebo)/resource/crazyflie_internal_model_controller.yaml" />
+   </node>
+   <!-- Enable/Disable the trajectory generator - If the position_controller is activated, the hovering_example will be executed,
+        otherwise the spline generator and the Mellinger's controller will be run-->
+   <node name="hovering_example_spline_swarm" pkg="rotors_gazebo" type="hovering_example_spline_swarm" output="screen" >
```

Comment:
only for the prove of concept?

Reply:
made some changes to use 'enable_mellinger_controller' arg for using trajectory instead of hovering but this code block will be removed after the drone wrapper is updated with updated HAL functioning